### PR TITLE
Add sub-campaign Clarinet

### PR DESCRIPTION
**Pull request by yu71ng**

### CODE REVIEW

**Comment by lkluft** [comment]
Hey Yuting, it looks like you got tangled up with your local branches. My current guess is that you created a the new branch `clarinet` from an existing branch with a new data policy instead of the `main` branch.

There are several ways to fix this. I would suggest that you perform an [interactive rebase](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase) from `clarinet` onto `main` and drop the `Add data policy` commit.

Let me know if you need any help :)

**Comment by yu71ng** [comment]
Alright... I tried to perform the interactive rebase... but I probably shouldn't force-pushed it, right? 😅

**Comment by lkluft** [comment]
Actually, you **must** force-push after a rebase, because you have changed the history of the branch. Everything is now clean and ready to merge. Thanks for the contribution and the cleanup! :)

File: orcestra_book/clarinet.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+## CLARINET
```
Comment (lkluft) [inline orcestra_book/clarinet.md:1]: I think the heading in a document need to start on level H1 (e.g. `# CLARINET`)

**Review by lkluft — CHANGES_REQUESTED** [review]

File: orcestra_book/clarinet.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+## CLARINET 
+### CLoud and Aerosol Remote sensing for EarThcare
```
Comment (lkluft) [inline orcestra_book/clarinet.md:2]: Headings in a document need to start on level H1, i.e.:
```suggestion
# CLARINET
## CLoud and Aerosol Remote sensing for EarThcare
```

**Review by lkluft — CHANGES_REQUESTED** [review]

**Comment by yu71ng** [comment]
I see... well... learning by doing 😅

**Review by lkluft — APPROVED** [review]
Thanks for your contribution! :)

**Pull request merged**